releasenotes/notes/oracle-reconnect-on-connection-lose-before-select-54d2b6ad6811d1ea.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix nil pointer error on Oracle DBM query when check connection is lost before SELECT.
```

Would it be correct to say:

```suggestion
    Fix nil pointer error on Oracle DBM query when the check's connection is lost before SELECT statement executes.
```

Or something like that? I think "check connection" and "SELECT" need more information